muxpi/rpi/flash_rpi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright (C) 2019 Luxoft Sweden AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+# This flashes the image and tests it on RPI. 
+# This script must be run on muxpi.
+# Usage: 
+# $ ./flash_rpi.sh DUT_IP IMAGE
+#    DUP_IP = a string with IP, can be an empty string
+#    IMAGE = a string with the name of the variant
+
+set -e
+
+DUT_IP="$1"
+VARIANT="$2"
```

Comment:
Rename VARIANT to image. Variant should only be used to determine if its an intel, intel_qtauto, rpi, or rpi_qtauto build. In this case, we use it as an image.

Reply:
I see that this happens on other files as well.. Its confusing, but its up to you if you want to do the change or not.